euphonic/qpoint_frequencies.py

```diff
@@ -89,7 +90,9 @@ def __setattr__(self, name: str, value: Any) -> None:
 
     def calculate_dos(self, dos_bins: Quantity,
                       mode_widths: Optional[Quantity] = None,
-                      mode_widths_min: Quantity = Quantity(0.01, 'meV')
+                      mode_widths_min: Quantity = Quantity(0.01, 'meV'),
+                      adaptive_method: Optional[str] = 'default',
+                      error: Optional[float] = 0.001,
```

Comment:
Perhaps this should be 'adaptive_error' to link it more obviously to adaptive broadening.